channel/conclude.go

```diff
@@ -98,6 +98,70 @@ func (a *Adjudicator) ensureConcluded(ctx context.Context, req channel.Adjudicat
 	}
 }
 
+// checkConcludedState checks whether the concluded state is equal to the
+// expected state.
+func (a *Adjudicator) checkConcludedState(
+	ctx context.Context,
+	req channel.AdjudicatorReq,
+	subStates channel.StateMap,
+) error {
+	states := channel.MakeStateMap()
+	states.Add(req.Tx.State)
+	for _, v := range subStates {
+		states.Add(v)
+	}
+
+	events := make(chan *subscription.Event, adjEventBuffSize)
```

Comment:
Maybe add a comment here why we need to subscribe for specific events at this point.